aaq/tests/test_views.py

```diff
@@ -527,3 +527,88 @@ def test_response_feedback_invalid_feedback_text_view(self):
         assert response.json() == {
             "feedback_sentiment": ['"test" is not a valid choice.']
         }
+
+
+class CheckUrgencyV2ViewTests(APITestCase):
+    url = reverse("aaq-check-urgency-v2")
+
+    @responses.activate
+    def test_urgency_check_urgent(self):
+        """
+        Test that we can get an urgency score of 1.0
+        """
+        user = get_user_model().objects.create_user("test")
+        self.client.force_authenticate(user)
+        fakeAaqUdV2Api = FakeAaqUdV2Api()
+        responses.add_callback(
```

Comment:
Can you validate the request payload that is sent to the AAQ endpoint please?

Reply:
i have this test test_urgency_check_invalid to check if the payload is valid

Reply:
That is the payload to our endpoint, I'm talking about the payload e send to the aaq endpoint